include/xrpl/protocol/TER.h

```diff
@@ -339,7 +340,12 @@ enum TECcodes : TERUnderlyingType {
     tecINVALID_UPDATE_TIME = 188,
     tecTOKEN_PAIR_NOT_FOUND = 189,
     tecARRAY_EMPTY = 190,
-    tecARRAY_TOO_LARGE = 191
+    tecARRAY_TOO_LARGE = 191,
+    tecMPTOKEN_EXISTS = 192,
+    tecMPT_MAX_AMOUNT_EXCEEDED = 193,
+    tecMPT_LOCKED = 194,
+    tecMPT_NOT_SUPPORTED = 195,
+    tecMPT_ISSUANCE_NOT_FOUND = 196
```

Comment:
I am fine with all the suggestions above except changing `tecMPT_LOCKED`.  Ultimately we should not confuse locking with freeze.

Reply:
I just read [your comment below](https://github.com/XRPLF/rippled/pull/5143#discussion_r1784686883), and I (reluctantly) agree that "frozen" and "locked" are just different that reusing "frozen" in the API could cause confusion.

However, I have two followups:
1. Does that mean that if an account sets the account-level global freeze flag, that MPTs are _not_ altered / locked? Should they be?
2. Rename `tecMPT_LOCKED` to `tecLOCKED`, so it can be reused in the future with other concepts.

Reply:
1. MPT should have no affiliation with any existing flag that says "freeze"
2. will change